packages/react-components/src/components/SendBox.tsx

```diff
@@ -155,7 +146,15 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
             }
             onTyping && onTyping();
           }}
-          styles={concatStyleSets(TextFieldStyleProps, { fieldGroup: styles?.textField })}
+          styles={concatStyleSets(
```

Comment:
pull out the style concat function into a const as per https://github.com/microsoft/fluentui/wiki/Performance#good-1

Reply:
Good point. Thanks.